src/components/Avatar/AvatarMenu.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import StyleObjectPropType from '../../prop-types/style';
+import AvatarMenuItem from './AvatarMenu/AvatarMenuItem';
+import AvatarMenuNav from './AvatarMenu/AvatarMenuNav';
+import { AvatarMenuPosition } from './AvatarMenu/AvatarMenuEnums';
+import Avatar from './Avatar';
+import { AvatarSize } from './AvatarEnums';
+import './AvatarMenu.scss';
+
+const propTypes = {
+    avatar: PropTypes.instanceOf(Avatar).isRequired,
+    className: PropTypes.string,
+    children: PropTypes.arrayOf(PropTypes.element).isRequired,
+    onMenuItemClick: PropTypes.func,
+    open: PropTypes.bool,
+    position: PropTypes.string,
+    style: StyleObjectPropType,
+};
+
+const defaultProps = {
+    className: null,
+    onMenuItemClick: null,
```

Comment:
This could be set to `() => {}` which would remove the need for L41 I think

Reply:
Agreed. Although on this front I'd been copying the style we've used throughout UI React.  @jfrej what do you think?  If we change to `() => {}` here I feel we should do it everywhere

Reply:
We used to do that and there was a reason we switched to using `null`, but I can't remember it at the moment.
I agree that if we changed it here, we should change it everywhere. So for now I would keep the `null` and consider this change in a separate PR.

Reply:
Cool we'll stick with `null` for now then :)